must-gather/gather-olm-info

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+source ${SCRIPT_DIR}/common
+
+BASE_COLLECTION_PATH=$1
+LOGFILE_PATH="${BASE_COLLECTION_PATH}/${LOGFILE_NAME}"
+
+OLM_INFO_DIR=$BASE_COLLECTION_PATH/olm-info
+mkdir -p $OLM_INFO_DIR
```

Comment:
can we move all side-effects of running as script to start from main?

Reply:
done